build/NugetValidator/Program.cs

```diff
@@ -0,0 +1,55 @@
+﻿using CommandLine;
+using NuGet.Common;
+using NuGet.Protocol;
+using NuGet.Protocol.Core.Types;
+using NuGet.Versioning;
+
+namespace NugetValidator
+{
+    public class Options
+    {
+        [Option('n', "name", Required = true, HelpText = "NuGet Package Name")]
+        public string Name { get; set; }
+
+        [Option('v', "version", Required = true, HelpText = "Package Version")]
+        public string Version { get; set; }
+    }
+
+    internal class Program
```

Comment:
Could we add a little bit of error reporting, so we know if it's an issue with e.g. NuGet or the version string?